share/examples/bitcoin.conf

```diff
@@ -70,8 +70,8 @@
 # server=1 tells Bitcoin-Qt and bitcoind to accept JSON-RPC commands
 #server=0
 
-# Bind to given address to listen for JSON-RPC connections. Use [host]:port notation for IPv6.
-# This option can be specified multiple times (default: bind to all interfaces)
+# Bind to given address to listen for JSON-RPC connections.
+# Refer to the manpage or bitcoind -help for further details.
```

Comment:
This is true for all options, really, I've sometimes wondered if we shouldn't generate the example configuration somehow instead of having a separate place with options documentation here.